stacks/api/bucket/history/watcher.spec.ts

```diff
@@ -300,4 +310,27 @@ describe("Watcher", () => {
 
     expect(insertOneSpy).toHaveBeenCalledTimes(0);
   }, 35000);
+
+  it("shouldn't insert history if bucket history is false", async () => {
+    let disabledHistoryBucket = {...buckets[0], history: false};
+
+    schemaSpy.and.returnValue(Promise.resolve(disabledHistoryBucket));
+
+    let updatedDocument = {
+      ...firstBucketDocuments[0],
+      title: "updated title",
+      description: "updated description"
+    };
+
+    await databaseService
+      .collection(`bucket_${buckets[0]._id}`)
+      .replaceOne({_id: updatedDocument._id}, updatedDocument);
+
+    await new Promise(resolve => setTimeout(resolve, 100));
```

Comment:
This may cause this test to be flaky. 

Could you please use stream helpers from the `@spica-server/database/testing` module?